pos_picking_load/static/src/xml/pos_picking_load.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<template>
+
+    <t t-name="LoadPickingButtonWidget">
+        <span class="control-button order-split" t-att-style='widget.is_visible() ? "display: block;" : "display: none;"'>
+            <i class="fa fa-upload"></i>
+            <t t-esc="widget.button_text()"/>
+        </span>
+    </t>
+
+    <t t-name="LoadPickingLine">
+        <tr class="picking-line" t-att-data-picking-id="picking.id">
+            <td><t t-esc="picking.name"/></td>
+            <td><t t-esc="picking.partner_id[1] || ''"/></td>
+            <td><t t-esc="picking.min_date || ''"/></td>
+            <td><t t-esc="picking.origin || ''"/></td>
+        </tr>
+    </t>
+
+    <t t-name="LoadPickingScreenWidget">
+        <div class="pickinglist-screen screen">
+            <div class="screen-content">
+                <section class="top-content">
+                    <span class="button back picking-button">
+                        <i class="fa fa-times"></i>
+                        Cancel
+                    </span>
+                    <span class="button validate picking-button highlight">
+                        <i class="fa fa-arrow-circle-o-right"></i>
+                        Select
+                    </span>
+                    <span class="searchbox">
+                        <input placeholder="Search Picking" />
+                    </span>
+                </section>
+                <section class="full-content">
+                    <div class="window">
+                        <section class="subwindow collapsed">
```

Comment:
This doesn't seem to be needed, it pushes the selection list down.
![pos_picking_load](https://user-images.githubusercontent.com/4568032/60981828-5b315800-a337-11e9-9aa0-3ed9459167ff.png)

Reply:
Hi @kirca ! 

Well, I copy pasted windows present in Odoo. (partner selection).

On runbot, in my computer, it works like a charm.

![pos_picking_load](https://user-images.githubusercontent.com/3407482/60983800-e6f8b380-a33a-11e9-9223-8a4cb038456b.gif)

So, I guess it's browser trouble don't you think ? 

for my Side, Mozilla Firefox for Ubuntu, 67.0.4 (64 bits)

do you face the same problem, when you select a customer ?

thanks !

Reply:
Indeed, issue is with Chrome. I did some digging and found commit where there is fix for it in the customer selection screen https://github.com/odoo/odoo/commit/4ff34a98bcefa96c908baae1646a5af8acd21913#diff-401e9bb2837c2ac0b281c1c8a9c195ceR440 . Just need to add "collapsed" class next to "subwindow-container".

Ubuntu 16.04
Chrome - Version 72.0.3626.119 (Official Build) (64-bit)
Firefox - 67.0.4 (64-bit)